### DISCUSSION TITLE
Cropper area is initially very small

### DISCUSSION THREAD

**Discussion by kschmott** [ValentinH/react-easy-crop#398] [Q&A] [answered]
Hello all,
I was using this amazing cropper but I just have one issue with it. For my app, a user will click on a preview and then a modal appears with the cropper on it. The only problem is that the cropper area is initially very small compared to the image. Only after changing the aspect ratio in state does the cropper fit the cropped area to the width of the image. Any thoughts on this?

**Comment by ValentinH** [comment, marked as answer]
Usually for this kind of issues, you need to make sure that the Cropper is only rendered once the modal is open.

**Reply by kschmott** [reply]
Ahh so if there is an animation it could mess with it

**Reply by ValentinH** [reply]
Yes this is an inconvenient. To workaround this, you could provide a placeholder when the modal is not used. A good placeholder could be the image to be cropped. I did this in a previous company and it worked well. Basically, when opening only the image was shown and then the crop UI was appearing on top of it.
